extensions/wikia/Search/classes/Services/CombinedSearchService.php

```diff
@@ -213,182 +266,215 @@ public function queryPhraseForWikias( $query, $hubs, $lang ) {
 		$select->createFilterQuery( 'users' )->setQuery( 'activeusers_i:[0 TO *]' );
 		$select->createFilterQuery( 'pages' )->setQuery( 'articles_i:[50 TO *]' );
 		//speedydeletion: 547090, scratchpad: 95, lyrics:43339,
-		$select->createFilterQuery( 'banned' )->setQuery( '-(id:547090) AND -(id:95) AND -(id:43339) AND -(hostname_s:*answers.wikia.com)' );
+		$select->createFilterQuery( 'banned' )->setQuery(
+			'-(id:547090) AND -(id:95) AND -(id:43339) AND -(hostname_s:*answers.wikia.com)'
+		);
 
 		$dismax->setBoostQuery( 'domains_txt:"www.' . preg_replace( '|\s*|', '', $phrase ) . '.wikia.com"^1000' );
 		$dismax->setBoostFunctions( 'wam_i^5 articles_i^0.5' );
 
 		$result = $client->select( $select );
+
 		return $this->extractData( $result, $fields );
 	}
 
 	protected function prepareXWikiQuery( $query, $hubs, $lang ) {
 		$hubsQuery = '';
 		if ( !empty( $hubs ) ) {
-			$hubsList = [ ];
+			$hubsList = [];
 			foreach ( $hubs as $hub ) {
-				$hubsList[ ] = '(hub_s:' . $hub . ')';
+				$hubsList[] = '(hub_s:' . $hub . ')';
 			}
 			$hubsQuery = '+(' . implode( ' OR ', $hubsList ) . ') AND ';
 		}
-		return $hubsQuery . '+(lang_s:' . $lang .
-		') AND +((sitename_txt:"' . $query . '") OR (headline_txt:"' . $query . '") OR (description_txt:"' . $query .
-		'") OR (domains_txt:"www.' . preg_replace( '|\s*|', '', $query ) . '.wikia.com"))';
+
+		return $hubsQuery .
+		'+(lang_s:' .
+		$lang .
+		') AND +((sitename_txt:"' .
+		$query .
+		'") OR (headline_txt:"' .
+		$query .
+		'") OR (description_txt:"' .
+		$query .
+		'") OR (domains_txt:"www.' .
+		preg_replace( '|\s*|', '', $query ) .
+		'.wikia.com"))';
 	}
 
 	protected function extractData( $searchResult, $requestedFields ) {
-		$result = [ ];
+		$result = [];
 		foreach ( $searchResult->getDocuments() as $doc ) {
-			$item = [ ];
+			$item = [];
 			foreach ( $requestedFields as $key => $field ) {
 				$keyValue = !is_numeric( $key ) ? $key : $field;
 				$value = $doc->{$field};
 				if ( isset( $value ) ) {
-					$item[ $keyValue ] = $value;
+					$item[$keyValue] = $value;
 				}
 			}
-			$result[ ] = $item;
+			$result[] = $item;
 		}
+
 		return $result;
 	}
 
 	protected function prepareQuery( $query, $namespaces, $lang, $hubs = null, $minArticleQuality = null ) {
-		$nsArr = [ ];
+		$nsArr = [];
 		$hubQuery = '';
 		if ( !empty( $hubs ) ) {
-			$hubsArr = [ ];
+			$hubsArr = [];
 			foreach ( $hubs as $hub ) {
-				$hubsArr[ ] = "(hub:$hub)";
+				$hubsArr[] = "(hub:$hub)";
 			}
 			$hubQuery = '(' . implode( ' OR ', $hubsArr ) . ') AND ';
 		}
 		foreach ( $namespaces as $ns ) {
-			$nsArr[ ] = "(ns:$ns)";
+			$nsArr[] = "(ns:$ns)";
 		}
-		$query = '+(' . $hubQuery . '(' . implode( ' OR ', $nsArr ) . ') AND (lang:' . $lang .
-			')) AND +((title_en:"' . $query . '") OR (redirect_titles_mv_en:"' . $query . '")) AND +(nolang_txt:"' . $query . '")';
+		$query = '+(' .
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$query". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)